Patterns/Italian/Italian-Numbers.yaml

```diff
@@ -1,41 +1,45 @@
-  # TODO: Work on this one
-  def: (({ZeroToNineIntegerRegex}(\s+cento))|cento|((\s+cento\s)+{TensNumberIntegerRegex}))
+  def: (settanta?|venti?|trenta?|ottanta?|novanta?|quaranta?|cinquanta?|sessanta?)
+HundredsNumberIntegerRegex: !nestedRegex #100,200,300,...900
+  def: ({ZeroToNineIntegerRegex}?cento?)
   references: [ ZeroToNineIntegerRegex, TensNumberIntegerRegex ]
-BelowHundredsRegex: !nestedRegex
-  def: (({TenToNineteenIntegerRegex}|({TensNumberIntegerRegex}(\W+{ZeroToNineIntegerRegex})?))|{ZeroToNineIntegerRegex})
-  references: [ TenToNineteenIntegerRegex, TensNumberIntegerRegex, ZeroToNineIntegerRegex ]
-BelowThousandsRegex: !nestedRegex
-  def: (({HundredsNumberIntegerRegex}(\s+{BelowHundredsRegex})?|{BelowHundredsRegex}|{TenToNineteenIntegerRegex})|cento\s+{TenToNineteenIntegerRegex})
-  references: [ HundredsNumberIntegerRegex, BelowHundredsRegex, TenToNineteenIntegerRegex ]
-SupportThousandsRegex: !nestedRegex
-  def: (({BelowThousandsRegex}|{BelowHundredsRegex})\s+{RoundNumberIntegerRegex}(\s+{RoundNumberIntegerRegex})?)
-  references: [ BelowThousandsRegex, BelowHundredsRegex, RoundNumberIntegerRegex ]
+BelowHundredsRegex: !nestedRegex #0,1,2,...10,11,12,...,20,21,22,...,98,99
+  def: ({ZeroToNineIntegerRegex}|{TenToNineteenIntegerRegex}|({TensNumberIntegerRegex}({ZeroToNineIntegerRegex})?))
+  references: [ ZeroToNineIntegerRegex, TenToNineteenIntegerRegex, TensNumberIntegerRegex]
+BelowThousandsRegex: !nestedRegex #100,101,102,...210,211,212,...998,999
+  def: ({HundredsNumberIntegerRegex}{BelowHundredsRegex}?)
+  references: [ HundredsNumberIntegerRegex, BelowHundredsRegex]
+BelowThousandsTotRegex: !nestedRegex #0,1,2,...,998,999
+  def: ({BelowHundredsRegex}|{BelowThousandsRegex})
+  references: [ BelowThousandsRegex, BelowHundredsRegex]
+BelowMillionsRegex: !nestedRegex #1000,1001,...999.998,999.999
+  def: ((({BelowHundredsRegex}|{BelowThousandsRegex})mila)|mille)((\s+e\s+)?({BelowHundredsRegex}|{BelowThousandsRegex}))?
+  references: [ BelowHundredsRegex, BelowThousandsRegex]
+BelowMillionsTotRegex: !nestedRegex #0,1,2,...999.998,999.999
+  def: ({BelowHundredsRegex}|{BelowThousandsRegex}|{BelowMillionsRegex})
+  references: [ BelowHundredsRegex, BelowThousandsRegex, BelowMillionsRegex]
 NegativeNumberTermsRegex: !simpleRegex
-   # TODO: modify below regex according to the counterpart in English
-  def: ^[.]
+  def: (meno\s+)
 NegativeNumberSignRegex: !nestedRegex
-  def: ^({NegativeNumberTermsRegex}\s+).*
+  def: ^{NegativeNumberTermsRegex}.*
   references: [ NegativeNumberTermsRegex ]
-SeparaIntRegex: !nestedRegex
-  def: ({SupportThousandsRegex}(\s+{SupportThousandsRegex})*(\s+{BelowThousandsRegex})?|{BelowThousandsRegex})
-  references: [ SupportThousandsRegex, BelowThousandsRegex ]
-AllIntRegex: !nestedRegex
-  def: ({SeparaIntRegex}|mille(\s+{BelowThousandsRegex})?)
-  references: [ SeparaIntRegex, BelowThousandsRegex ]
+SeparaIntRegex: !nestedRegex #0,1,...,1.000.000,2.000.000,...,999.999.000.000,...,
```

Comment:
Feel free to add such comments in the English YAML too if they improve readability and understanding of the patterns.